android/filament-android/src/main/cpp/Material.cpp

```diff
@@ -279,6 +279,16 @@ Java_com_google_android_filament_Material_nHasParameter(JNIEnv* env, jclass,
     return (jboolean) hasParameter;
 }
 
+extern "C" JNIEXPORT jstring JNICALL
+Java_com_google_android_filament_Material_nGetParameterTransformName(JNIEnv* env, jclass,
+        jlong nativeMaterial, jstring samplerName_) {
+    Material* material = (Material*) nativeMaterial;
+    const char* samplerName = env->GetStringUTFChars(samplerName_, 0);
+    jstring transformName_ = env->NewStringUTF(material->getParameterTransformName(samplerName));
```

Comment:
`getParameterTransformName` could return `nullptr` if the samplerName didn't exist. In that case, you need to return an empty string according to the javadoc.